src/processTargets/targets/InteriorTarget.ts

```diff
@@ -0,0 +1,22 @@
+import { shrinkRangeToFitContent } from "../../util/selectionUtils";
+import BaseTarget, { CommonTargetParameters } from "./BaseTarget";
+
+export default class InteriorTarget extends BaseTarget {
+  insertionDelimiter = "";
+
+  constructor(parameters: CommonTargetParameters) {
+    super(parameters);
+  }
+
+  get contentRange() {
+    return shrinkRangeToFitContent(this.editor, this.state.contentRange);
```

Comment:
huh.  It's a bit confusing that the `contentRange` is actually the `removalRange` for this target 🤔.  Not sure I have a better solution, but it's a bit surprising.  Maybe `InteriorTarget` is the exception to `CommonTargetParameters` containing `contentRange`?

Let's Discord this one when you're back

Reply:
I'd be tempted to have something like `definingRange` as the common parameter, and then `contentRange` by default returns it, but things can override it

Reply:
We can do that

Reply:
You want me to go ahead and do that change in this pr?

Reply:
Might be worth a quick convo

Reply:
Sounds good